data/presets/presets/indoor/elevator.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon":"temaki-elevator",
+    "fields": [
+        "level",
+        "name"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "indoor": "room",
+        "highway": "elevator"
```

Comment:
Hmm using `highway=elevator` on areas can be complicated for routers to handle. Any chance we could use `elevator=yes` for this? This would be like how we map `highway=corridor` separately from `indoor=corridor`.

Reply:
@quincylvania, I followed the Simple Indoor Tagging scheme as documented as closely as possible. I agree with some of your comments related to the exact wording. However, I'm not sure if we want to deviate from the official recommendation. There have been a few iterations on the scheme and it seems to have stabilised around this.

Link to the details on the connections between levels:
https://wiki.openstreetmap.org/wiki/Simple_Indoor_Tagging#Modelling_connections_between_different_levels

Reply:
Or maybe a non-searchable tag would work for now? Then at least we'll have the symbols.

Reply:
> Since these elevators will always be inside, they will never be connected to another way and therefore also shouldn't be a direct issue with routing.

The issue here is that elevators should be connected to other highways so routers can use them for indoor routing. This is relevant even before detailed indoor routing. For example, wheelchair users in hilly cities will often enter a building, take an elevator up, and exit on a higher level rather than climb a hill.

> Apart from this, any recommendations on how to proceed if we would actually like to modify this tagging scheme?

Looking more into this, there are only [about 380 ways](https://overpass-turbo.eu/s/MxZ) with `highway=elevator` and `indoor=room` right now. There's currently no great method of changing tagging consensus, but with so few uses perhaps this can be accomplished via the tagging mailing list…

> Or maybe a non-searchable preset would work for now? Then at least we'll have the symbols.

I don't really want iD mappers to see this preset in the wild, have it look sort of official, and wonder why they can't add it themselves.

Reply:
> The issue here is that elevators should be connected to other highways so routers can use them for indoor routing. This is relevant even before detailed indoor routing. For example, wheelchair users in hilly cities will often enter a building, take an elevator up, and exit on a higher level rather than climb a hill.

I'm a bit confused now. You're initial argument was that with this scheme, we would tag a a closed way as a highway which might confuse routers. Now you say that the elevators should be highways but in fact have to be connected to other highways? Not to say that I don't agree with your arguments about routing.

In order to facilitate the discussion on changing the indoor tagging scheme, what would you suggest to change?

Reply:
Ah, apologies for the confusion. My contention is that `highway=elevator` should only be used on nodes along the footway network and not on ways at all. 

<img width="549" alt="Screen Shot 2019-09-25 at 10 41 03 AM" src="https://user-images.githubusercontent.com/2046746/65584514-4d2bc480-df81-11e9-8b3b-b1280d69931e.png">

I suggest switching to `elevator=yes`, which means the thing (the room) _has_ an elevator, while `highway=elevator` means it _is_ an elevator. In fact, I'm just going to merge this and make the change myself. It seems like a valid use of an existing tag.